controllers/Alerts.js

```diff
@@ -1,6 +1,9 @@
 const AlertRouter = require('express').Router();
 const mongodb = require('mongodb');
 const Alert = require('../models/Alert');
+const User = require('../models/User');
+require('dotenv').config();
```

Comment:
Do we need this here?   It's already being called in `app.js` so not sure if we need to call it twice even though it's in a diff file